jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/applications/AliasEndpoint.java

```diff
@@ -594,4 +594,19 @@ public Response deleteDestructive(@PathParam("id") String id, @Context HttpServl
 			return appendAllowOriginHeader(Response.status(Status.INTERNAL_SERVER_ERROR), request);
 		}
 	}
+
+	@POST
+	@Path("/kc/updateGuids")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	@ReturnType("org.jboss.aerogear.unifiedpush.api.Alias")
+	public Response updateKcGuids(@Context HttpServletRequest request) {
```

Comment:
I don't see how it `@Consumes(MediaType.APPLICATION_JSON)` ? Is there any posted payload ? If not, then definitely not `@Consumes(MediaType.APPLICATION_JSON)`. If yes - where is the payload ?